app/views/sessions/_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= bootstrap_form_for :user do |f| %>
```

Comment:
Scott, I got the modal to post to the appropriate path using the following code:
```
<%= bootstrap_form_for :user, url: '/signup' do |f| %>
```